finance-api/api-reference.md

```diff
@@ -0,0 +1,288 @@
+Finance API reference
+===
+## About Finance API
+The Finance API is used to fetch information about the transactions that are made through Zettle. For example, card payments made with Zettle card readers.
+
+* [About Finance API](#about-finance-api)
+  * [Base URL](#base-url)
+  * [OAuth scope](#oauth-scope)
+* [Fetch account balance](#fetch-account-balance)
+  * [Parameters](#parameters)
+  * [Responses](#responses)
+* [Fetch account transactions](#fetch-account-transactions)
+  * [Parameters](#parameters)
+  * [Responses](#responses)
+* [Fetch payout information](#fetch-payout-information)
+  * [Parameters](#parameters)
+  * [Responses](#responses)
+* [Examples](#examples)
+  * [Fetch current balance for a liquid account](#fetch-current-balance-for-a-liquid-account)
+  * [Fetch transactions for a liquid account](#fetch-transactions-for-a-liquid-account)
+  * [Fetch payout information on a specific day](#fetch-payout-information-on-a-specific-day)
+* [Related resources](#related-resources)
+* [Related API reference](#related-api-reference)
+  
+### Base URL
+https://finance.zettle.com
+
+### OAuth Scope
+`READ:FINANCE`
+<!-- For more information on how to get authorisaition for the scope, see See [OAuth2 API](https://github.com/iZettle/api-documentation/blob/master/authorization.adoc) -->
+
+## Fetch account balance
+Returns the balance that is available in a merchant's Zettle preliminary or liquid account at a specific time.
+
+After a payment has been made from a merchant's liquid account to their bank account (daily, monthly or weekly), the balance is usually zero or the minimum balance in the merchant's Zettle account. In other cases, you can see how much money is in the merchant's Zettle account.
```

Comment:
We refer to this as  a payout in other cases not a "payment". 
Also in case of PayPal payment the money will be paid out to merchant's PayPal wallet.